WordPress/gutenberg
# Increase tap-target of primary action on unsupported blocks

## Description
Based on the issue: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2567#
this PR introduces an increase of tap-target of primary action on unsupported blocks

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2608

## How has this been tested?
1. Open Gutenberg mobile editor
2. Add unsupported block
3. Try to click on unsupported block surface
4. The options sheet should be presented

## Checklist:
